packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/UpdateSliceZoneModal.tsx

```diff
@@ -44,6 +44,7 @@ const UpdateSliceZoneModal: React.FC<UpdateSliceModalProps> = ({
         title: "Update Slice Zone",
       }}
       omitFooter={!projectHasAvailableSlices}
+      dataCy="update-slices-modal"
```

Comment:
Is there a reason we use `data-cy` everywhere but here?

Reply:
`dataCy` is a property name so that it use `data-cy` on the actual element.